raft/raft.go

```diff
@@ -54,6 +54,13 @@ type Progress struct {
 	Match, Next uint64
 	Wait        int
 	Unreachable bool
+	// If there is a pending snapshot (pendingSnapshot != 0), the replication
+	// process of this Progress will be paused. raft will not resend snapshot
+	// until the pending one is reported to be failed.
+	// PendingSnapshot is set when raft sends out a snapshot to this Progress.
```

Comment:
Mention somewhere in this comment that the numeric value is the log index of the snapshot that was sent.